### PR TITLE
Update spatiald version

### PR DESCRIPTION
**Pull request by oblm**
**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
This updates to the [latest](https://improbable.slack.com/archives/C0KP2A1UM/p1579532135001600) spot version.

#### Release note
Added to CHANGELOG.md.

#### Tests
**WIP** do not merge: I'm running Example Project CI against this now.

#### Documentation
None.

#### Primary reviewers
@joshuahuburn

### CODE REVIEW

**Comment by oblm** [comment]
Don't approve this until I provide a passing CI build but when you do, also approve these two PRs that duplicate this into `0.6.4-rc` & `0.8.1-preview-rc`.

**Comment by oblm** [comment]
Example Project CI running here: https://buildkite.com/improbable/unrealgdkexampleproject-nightly/builds/414

File: SpatialGDK/Extras/spot.version [hunk 1]
```diff
@@ -1 +1 @@
-20191029.144741.87a7d78768
+20200120.115350.8d6b779c82
```
Comment (StefaniaChierici) [inline SpatialGDK/Extras/spot.version]: this is the spot version, it is different from spatiald and they will not have the same versions.

Comment (StefaniaChierici) [reply SpatialGDK/Extras/spot.version]: I believe this is what needs changing! https://github.com/spatialos/UnrealGDK/blob/91c7984c222e062e02322e8be5fb3d53bc7ed867/SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp#L23

Comment (oblm) [reply SpatialGDK/Extras/spot.version]: that is a very good point, changing now

**Comment by oblm** [comment]
> Example Project CI running here: https://buildkite.com/improbable/unrealgdkexampleproject-nightly/builds/414

This CI run is failing due to: [UNR-2731](https://improbableio.atlassian.net/browse/UNR-2731)

In lieu of auto-testing I've set up a manual test run here: [IQA-819](https://improbableio.atlassian.net/browse/IQA-819)

**Comment by oblm** [comment]
Test run passed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased-`x.y.z`] - 2020-xx-xx
 
 ### Features:
+- Updated the version of the local API service used by the UnrealGDK.
```
Comment (joshuahuburn) [inline CHANGELOG.md]: Shouldn't this be a bugfix which contains something like "Fixed a bug which caused the local API service to memory leak"

Comment (oblm) [reply CHANGELOG.md]: Good point, I've added this in all three branches.

**Pull request merged**